tfhe/src/lib.rs

```diff
@@ -33,7 +32,20 @@
 #![allow(clippy::float_cmp)] // 7
 #![allow(clippy::bool_to_int_with_if)] // 6
 #![allow(clippy::unsafe_derive_deserialize)] // 1
-#![allow(clippy::cast_possible_wrap)] // 1
+#![allow(clippy::cast_possible_wrap)]
+// 1
```

Comment:
Shouldn't this comment be inlined with the previous statement?

Reply:
cargo fmt puts it there, I tried 🤷‍♂️

Reply:
Ok leave it as it is then